src/cascade/executor/data/parameters.toml

```diff
@@ -16,9 +16,13 @@ default = 1.0
 help = "Seconds between checks on whether subprocesses are done"
 
 [mvid]
-type = "str"
+type = "int"
```

Comment:
The mvid code above for argument_parser.py was making me wonder if the mvid in parameters.toml was updated to int, which I see it was here!

Reply:
I did this and Drew can complain if he wants but getting to the point where we support non-integer mvids will take a lot of work involving both our team and central comp so for now I think we need to treat them as integers.